internal/avgql/client.go

```diff
@@ -2,21 +2,29 @@ package avgql
 
 import (
 	"context"
+	"net/url"
 	"os"
 
+	"emperror.dev/errors"
 	"github.com/aviator-co/av/internal/config"
 	"github.com/shurcooL/graphql"
+	"github.com/sirupsen/logrus"
 	"golang.org/x/oauth2"
 )
 
-func NewClient() *graphql.Client {
+func NewClient() (*graphql.Client, error) {
 	src := oauth2.StaticTokenSource(
 		&oauth2.Token{AccessToken: config.Av.Aviator.APIToken},
 	)
 	httpClient := oauth2.NewClient(context.Background(), src)
 	apiURL := os.Getenv("AV_GRAPHQL_URL")
 	if apiURL == "" {
-		apiURL = "https://api.aviator.co/graphql"
+		baseURL, err := url.Parse(config.Av.Aviator.APIHost)
```

Comment:
should config take priority or should env var take priority?

Config taking prio makes sense to me.

Reply:
I think environment variables should usually take priority since it's easier to override them in more local contexts. My order is always:
* Defaults
* Config files
* Environment variables
* Flags